src/api/Controllers/LayerController.cs

```diff
@@ -61,6 +61,11 @@ public async Task<ActionResult<Layer>> GetByIdAsync(int id)
         return Ok(layer);
     }
 
+    /// <inheritdoc />
+    [Authorize(Policy = PolicyNames.Viewer)]
```

Comment:
Ich finde es immer noch sehr verwirrend mit den Permissions. Die "Viewer" permissions entsprechend ja im Client dem "Editor modus", richtig? Während der Viewer im Client nur auf "AllowAnonymous" endpoints kommt ??
Falls das so richtig ist, könnten wir nicht zumindest in unserem neuen Api das naming irgendwie anpassen?

Reply:
@MiraGeowerkstatt  Da stimme ich dir zu, jedoch müssten wir dabei bei den Permissions im Client ansetzen und von dort aus die Benennungen korrekt durchziehen oder am Besten aufräumen. Die _Viewer_-Permissions im API entsprechen allen Benutzern im Client, die **nicht** Admin sind.  
Wäre für die Policy ein Name wie z.B. `PolicyNames.NonAdmin` oder `PolicyNames.Editor` verständlicher? Wobei `Editor` wieder mit den Namen der Permission innerhalb einer Workgroup kollidiert und zu Verwechslungen führen könnte.

Reply:
Alles klar, vielleicht dann einfach als Issue ins Backlog. Und wenns mans anpackt gleich alles richtig durchziehen :)